demisto_sdk/commands/format/update_generic_json.py

```diff
@@ -34,7 +34,7 @@ def save_json_to_destination_file(self):
         if self.source_file != self.output_file:
             click.secho(f'Saving output JSON file to {self.output_file}', fg='white')
         with open(self.output_file, 'w') as file:
-            ujson.dump(self.data, file, indent=4)
```

Comment:
unfortunately, we should use ujson but it is missing some keys here - it should be:
```
ujson.dump(self.data, file, indent=4, encode_html_chars=True, escape_forward_slashes=False,
                   ensure_ascii=False)
```
the reason being `json` package can change the json file from what we currently have